packages/core/contracts/insured-bridge/BridgeDepositBox.sol

```diff
@@ -155,6 +141,8 @@ abstract contract BridgeDepositBox is Testable, Lockable {
         emit DepositsEnabled(l2Token, depositsEnabled);
     }
 
+    function bridgeTokens(address l2Token, uint32 l2Gas) public virtual;
```

Comment:
this is the virtual interface I added. this means that implementers only need to use `BridgeDepositBox` when calling methods and can ignore chain-specific interfaces.

Note that all other important methods in this file contain the same type of interface